packages/mantine/src/components/crud/create/index.tsx

```diff
@@ -64,6 +65,20 @@ export const Create: React.FC<CreateProps> = (props) => {
 
     const resource = resourceWithRoute(resourceFromProps ?? routeResourceName);
 
+    const { options } = useRefineContext();
+
+    const breadcrumb: ReactNode =
+        typeof breadcrumbFromProps === "undefined"
+            ? options?.breadcrumb
+            : breadcrumbFromProps;
+
+    const breadcrumbComponent =
+        typeof breadcrumb !== "undefined" ? (
+            <>{breadcrumb}</> ?? undefined
```

Comment:
```suggestion
            {breadcrumb}
```

Isn't the undefined check already done above?